### PR TITLE
Ztp fix

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by jeffmart — APPROVED** [review]

File: docs/intro_install_quickstart_otpiso.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Quickstart From the OTP ISO
+sidebar_label: Quickstartfrom the OTP ISO
```
Comment (MichaelBaj) [inline docs/intro_install_quickstart_otpiso.md:3]: Space needed between Quickstart and from

Inconsistent capitalization within document. It should be "QuickStart" throughout

File: docs/intro_install_quickstart_otpiso.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The 128T router will need to be provisioned on the conductor.  This procedure presumes you are familiar with the [concepts](concepts_glossary.md) and [configuration](config_basics.md) of the 128T platform.
+
+When a router configuration has been added to the conductor, but the device has not yet connected, in place of device-specific information, QuickStart instructions will be displayed.
+
+After the configuration has been added to the authority on the conductor:
+
+- On the Conductor, go to the UI to start the QuickStart process for the newly created 128T Router by accessing “Routers” -> “Router Name” -> “QUICKSTART LINK”
+
+![QuickStart Generate QuickStart Link](/img/intro_ztp_quickstart_server_2.png)
```
Comment (MichaelBaj) [inline docs/intro_install_quickstart_otpiso.md:18]: Is this a pre-existing image? It is not included in this PR.

File: docs/intro_install_quickstart_otpiso.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+dmidecode -t system | grep Serial
+```
+If you plan on setting the asset ID to the serial number, now is the time to make that change to the configuration on the conductor.
+:::
+
+The *device host address* is the IP address that is assigned to the 128T router during the staging process.  By default this is set to `192.168.0.128`.
+
+A password is used to encrypt the contents of the QuickStart file.  This password will be required when applying the file to the target platform.
+
+<p align="center"><img src="/img/intro_ztp_quickstart_server_3.png" alt="QuickStart Link Generation"/></p>
```
Comment (MichaelBaj) [inline docs/intro_install_quickstart_otpiso.md:36]: Why is this embedded HTML? This should not be necessary.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/intro_install_quickstart_otpiso.md (hunk 2 above)
Comment (jeffmart) [reply docs/intro_install_quickstart_otpiso.md:18]: It is, this was content that was accidentally removed when the ZTP aspects were removed.

File: docs/intro_install_quickstart_otpiso.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/intro_install_quickstart_otpiso.md]: done

**Review by MichaelBaj — APPROVED** [review]

File: docs/intro_install_quickstart_otpiso.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/intro_install_quickstart_otpiso.md]: The short answer is whoever wrote the topic originally wanted the graphic centered. I removed the HTML and put a standard link in, and the graphic left justified. No other impact.

**Pull request merged**